prober/http.go

```diff
@@ -313,7 +313,8 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 
 	httpConfig := module.HTTP
 
-	if !strings.HasPrefix(target, "http://") && !strings.HasPrefix(target, "https://") {
+	targetLower := strings.ToLower(target)
```

Comment:
we don't need to preserved target, so this can be written as `target := strings.ToLower(target)`

this avoids creating a new variable.

Reply:
I pushed this fix and rebased the PR.